etc/docker/test/matrix.yml

```diff
@@ -20,8 +20,6 @@ python:
         - client
         - client_syntax
   - "3.6"
-  - "3.7"
-  - "3.8"
   - "3.9"
```

Comment:
What about the 3.10?

Reply:
3.10 will need more work ;) New dockerfile's etc. That would be in another PR

Reply:
Aah btw, I just realized I need to make some more changes in the matrix_parser.py in order to enable pu37,py38 builds just for Postgres as db.